### DISCUSSION TITLE
Packaging app on linux

### DISCUSSION THREAD

**Discussion by not-a-doctor-stromberg** [edrlab/thorium-reader#1833] [Q&A]
Hi all,

I'm trying to build/install thorium on my machine. I ran `npm run package:linux`, but I'm not sure what the output of this is supposed to be. There didn't seem to be anything immediately obvious in the `dist` directory: I see some `index_library` and `index_pdf` and `preload.js` stuff, but I expected to see a `.deb` or `.appimage`. 

I looked into `electron-build` a little, and saw:
```
"linux": { "target": [ "deb", "AppImage"]}
```
in the `build` section of `package.json` -- I tried passing a `--target=AppImage` flag, but this did not seem to change anything. What output should I expect/where should it be?

**Comment by danielweck** [comment]
hello, check the ./release folder :)

**Reply by not-a-doctor-stromberg** [reply]
Thanks for the reply! I don't seem to have a release folder :(

Here's the logs of the build, immediately followed by `ls -la` and a `find`: https://pastebin.com/ng3vWWJU. Am I doing something else wrong?

**Reply by danielweck** [reply]
In your pastebin console log, the build process seems to end abruptly at line 395. Any idea?
